### ISSUE TITLE
is this usable?

### ISSUE THREAD

**Issue by jnny** [sudomesh/peoples-map#1]
hey @Juul - wondering about status of this repo and whether we should punt it to a higher priority or not?

i think we've stalled since last discussions / contention re: preserving privacy of node owners. have you thoughts / recollections of those convos, @paidforby / @jhpoelen / @eenblam / @bennlich ?

**Comment by bennlich** [comment]
Yeah--I started working on a new map, but I haven't given it a lot of attention yet. My privacy plan is to serve "approximate" locations unless the user logs in as admin, in which case you can see the real locations.

I can try to get something very basic out in the wild so we can start iterating on it. I think a map is a pretty important outreach tool, and could double as an important planning tool for future installations. 

I will probably wait until post-WINS before spending time on this tho.

**Comment by bennlich** [comment]
I also didn't know this repo existed :-P
